Plot language (#2239)
bootstrapworld/curriculum: lessons/Data-Science/correlations/langs/en-us/index.adoc

Let’s review what we have learned about correlations.

@QandA{

> @Q{What kind of visualizations can we use to visualize a correlation?}

Reviewer: ```suggestion
@Q{What kind of visualizations can we use to look for correlations?}
```
